packages/broker/src/ConfigWizard.ts

```diff
@@ -48,6 +48,11 @@ const logger = {
     }
 }
 
+function generateApiKey(){
+    const hex = uuid().split('-').join('')
+    return Buffer.from(hex).toString('base64')
```

Comment:
It would be good to generate alpha-numeric API keys. This can generate a key "ODUwMjg0YmVkMDMwNGNlYmIxOThlZGNhNzNiM2M2M2U="